### PR TITLE
Additional copy edits

### PR DESCRIPTION
**Pull request by jayrbolton**
These are my additional copy edits on those readme files.

I think the readmes under /data, /ui, /lib should not have instructions for the module creator. This ends up looking janky for existing modules when outside devs read the docs. I changed the wording so it feels more like general documentation.

### CODE REVIEW

File: src/java/us/kbase/templates/module_readme_data.vm.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-This directory should contain any data required for your module to work that is not already present in KBase.
-This includes both reference data for your module to run as well as data used in testing your code.
-If you have reference data that is too large to store on GitHub check out our [Reference Data Guide](https://kbase.github.io/kb_sdk_docs/howtos/work_with_reference_data.html)
\ No newline at end of file
+This directory contains any data required for this module to run that is not already present in KBase, including both reference data and test data.
```
Comment (JamesJeffryes) [inline src/java/us/kbase/templates/module_readme_data.vm.properties]: Actually, most modules we've written have the test data in `test/data`. Should have scrubbed that out myself

**Comment by JamesJeffryes** [comment]
Agreed. Just need to remove the instruction about storing test data in the `/data` directory

**Comment by jayrbolton** [comment]
I can do that in a second

**Comment by jayrbolton** [comment]
Okay, updated :man_cook:

**Review by JamesJeffryes — APPROVED** [review]
💯

**Pull request merged**